tests/test_models.py

```diff
@@ -364,6 +366,26 @@ def test_mars_local_tms():
     assert round(center.y, 6) == 17
 
 
+@pytest.mark.parametrize(
+    "identifier, url, crs", [
+        ("UPSAntarcticWGS84Quad", "https://raw.githubusercontent.com/vincentsarago/TileMatrixSets/master/UPSAntarcticWGS84Quad.json", 5042),
+        ("CanadianNAD83_LCC", "https://raw.githubusercontent.com/vincentsarago/TileMatrixSets/master/CanadianNAD83_LCC.json", 3978),
+        ("WebMercatorQuad", "https://raw.githubusercontent.com/vincentsarago/TileMatrixSets/master/WebMercatorQuad.json", 3857)
+    ]
+)
```

Comment:
better use local fixtures (to avoid fetching things from the internet)

Reply:
Changed!